SpywareFilter/sections/tracking_servers.txt

```diff
@@ -38,6 +38,7 @@
 !||i.snssdk.com/log/*
 !||i.snssdk.com/slardar/sdk.js
 !
+||check.tadapi.info^
```

Comment:
Please move it to the `tracking_servers_firstparty.txt`

Reply:
d340095180777f9c59b3a6657f6e359a219b31c5